x/claim/types/claim_record.go

```diff
@@ -23,3 +28,13 @@ func (claimRecord *ClaimRecord) IsEmpty() bool {
 
 	return true
 }
+
+// ChainFromString convert chain string to Chain Enum type
+func ChainFromString(chain string) (Chain, error) {
```

Comment:
this probably doesn't matter since we only have 2 chains and the iterative approach works, but I think we could easily use the mapping `Chain_value` to make this more efficient.  Not required, since i don't think the number of chains is gonna grow to make this not performant.

Reply:
changed it